editor/src/components/navigator/layout-element-icons.ts

```diff
@@ -181,6 +183,39 @@ function createLayoutIconProps(
   return null
 }
 
+function isConditionalBranchText(
+  navigatorEntry: NavigatorEntry | null,
+  metadata: ElementInstanceMetadataMap,
+): boolean {
+  function getElement(): JSXElementChild | null {
```

Comment:
I think this function should be renamed, because it's not getting "the element", but rather it's getting the element iff it's a synthetic navigator entry or the child of a conditional expression... 🤔